packages/util-crypto/src/nacl/convertKey.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017-2021 @polkadot/util-crypto authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable */
```

Comment:
Would not do this at all. Rather add a small `ed2curve.d.ts`

Reply:
nvm. there is a `@types/ed2curve`! Removing eslint-disable.